src/event/FalconEvent.ts

```diff
@@ -0,0 +1,41 @@
+export class FalconEvent {
```

Comment:
We could export this class as default

Reply:
We can try export default, but if it makes too many maintainability issues we will have to remove it